const/view_game.py

```diff
@@ -9,5 +9,9 @@
     "Weapon": {
         "center_x": (SCREEN_WIDTH / 5) + 500,
         "center_y": SCREEN_HEIGHT - 90
+    },
```

Comment:
We should be multiplying by global any time we set pixel values which should always be based on 1280x720.